### DISCUSSION TITLE
Is vueuse framework agnostic?

### DISCUSSION THREAD

**Discussion by deleonio** [vueuse/vueuse#401] [Q&A] [answered]
S. Title

**Comment by userquin** [comment, marked as answer]
See repo name: `vue`use.

FYI: I've seen some lab work from `lit-html` to make framework agnostic integrations: see this entry https://github.com/Polymer/lit-html/issues/1682

**Reply by deleonio** [reply]
I have seen the name "vue" but the usage looks native.

There is no couple visible. 

I think the pure way where at the end the best case.

Frameworks are in the most cases history

**Reply by dheimoz** [reply]
Hello @deleonio Vue use is based on Vue Composition API. Thanks.

**Reply by deleonio** [reply]
Okay, that means reusable code ... not more. The composition api is a default design pattern. So I think a lot of vueuse is framework agnostic. 

If I have time, I will check that with react and other frameworks...

**Reply by wheatjs** [reply]
VueUse uses Vue's reactivity system which is as far as I'm aware is not compatible with React. If you are looking for something that is compatible with React maybe checkout https://github.com/streamich/react-use

**Reply by antfu** [reply]
I am not sure if I got what you mean for "framework agnostic", but if you want to use it on React, yes you can: 

https://github.com/antfu/reactivue

Other than that, it's not really "framework agnostic" as it bounded to Vue's reactivity system.
